src/commands/compile/compileSingleFile.ts

```diff
@@ -36,17 +37,18 @@ export async function compileSingleFile(
 
   const sourcePath = path.isAbsolute(source)
     ? source
-    : path.join(process.projectDir, source)
+    : path.join(process.currentDir!, source)
 
   if (!(await validateSourcePath(sourcePath))) {
     throw new Error(`Provide a path to source file (eg '${commandExample}')`)
   }
 
+  const { buildDestinationFolder } = await getConstants()
   const outputPath = output
```

Comment:
compiled file should result in `jobs` or `services` folder in `sasjsbuild`

Reply:
fixed